packages/vite/src/node/plugins/html.ts

```diff
@@ -1510,7 +1511,7 @@ function serializeAttrs(attrs: HtmlTagDescriptor['attrs']): string {
     if (typeof attrs[key] === 'boolean') {
       res += attrs[key] ? ` ${key}` : ``
     } else {
-      res += ` ${key}=${JSON.stringify(attrs[key])}`
+      res += ` ${key}=${JSON.stringify(escapeHtml(attrs[key]))}`
```

Comment:
Checking [the example](https://github.com/component/escape-html#:~:text=console.dir(%27%3Cinput%20name%3D%22full_name%22%20value%3D%22%27%20%2B%20escapeHtml(fullName)%20%2B%20%27%22%3E%27)), I think it should be
```suggestion
      res += ` ${key}="${escapeHtml(attrs[key])}"`
```
otherwise, `JSON.stringify` will do unnecessary escapes.